test/ssr/ssr-string.spec.js

```diff
@@ -86,6 +86,87 @@ describe('SSR: renderToString', () => {
     })
   })
 
+  it('custom component style', done => {
+    renderVmWithOptions({
+      template: '<comp :style="style"></comp>',
+      data: {
+        style: 'color:red'
+      },
+      components: {
+        comp: {
+          template: '<div></div>'
+        }
+      }
+    }, result => {
+      expect(result).toContain(
+        '<div server-rendered="true" style="color:red"></div>'
```

Comment:
`<comp>` has a synthetic `server-rendered` attrs so it has the chance to execute module logic...